### ISSUE TITLE
Auto Content Refresh not working

### ISSUE THREAD

**Issue by AdiosOnline** [zammad/zammad#591]
Hi,

I don't think it is an Serverside Error, but maybe anybody has the same Issue or already found a Solution and could share it. 

I expect these Info Counter and "Animations" after new content on Tab automaticaly
![image](https://cloud.githubusercontent.com/assets/24896670/21682777/a20ee5fc-d356-11e6-9bb7-ab69e7d2a167.png)

### Infos:

* Used Zammad version: 1.1.1 (2016-12-14)
* Operating system: CentOS 7
* Browser + version: Chrome 55.0.2883.87, Firefox 50.1,

### Expected behavior:

* While working in Zammad, i should get notification, or refreshs in the Tickets.  The Notification Icon should count the new Events. 

### Actual behavior:

* I get no updates while I view Tickets or watch the Activity Stream. The "Notification Bird" doesn't change/count,  even when cklicking on it there are no new Elements. Reload the Page with F5 Displays the counter at the Notification Icon and Elements in open Tabs get updated.

### Steps to reproduce the behavior:

*

**Comment by AdiosOnline** [comment]
I'm still searching for a solution. Can anybody explain how this feature ist working? I guess the Zammad Server gives a push notification to the Client browser? May the Firewall block a socket connection and create my Issue? Thanks for any tips and advise...

**Comment by thorsteneckel** [comment]
Hi @AdiosOnline - That's it. Zammad has an open WebSocket connection to push events to to the browser client. This is used to inform the user about information regarding tickets and stuff. Please check your browser console in the "Network" tab and check if there are any issues with the Websocket connection.

It would be very helpful to be able to reproduce this behavior. Can you give us some insights about it?

**Comment by AdiosOnline** [comment]
I would be glad to provide you with any Information you need. Maybe we can get together by Phone to initiate a remote session, if needed. 

I dont understand what you mean , what I should check. I have tested my socket connections under http://www.websocket.org/echo.html with following results:
**Unsecured connection:**

`ERROR: undefined

DISCONNECTED

**secured connection:** Use secure WebSocket (TLS)

 CONNECTED

 SENT: Rock it with HTML5 WebSocket

 RECEIVED: Rock it with HTML5 WebSocket

 DISCONNECTED`

**Comment by AdiosOnline** [comment]
Hi @thorsteneckel ,

I found it! After your Information about the Socket Method, I focused on the Traffic betwen Server Client, and not Settings on the Client. 

The Solution is as easy as small.. We use Kaspersky on every Client, and my PC don't uses the Administrative policies. So on my PC, the new Domain suffix we use for Zammad, was not registered in exceptions from WEB-AntiVirus Module.

I hope this Information is useful for anybody.
